### ISSUE TITLE
Plugin returns OK when Elasticsearch cluster is "not there"

### ISSUE THREAD

**Issue by Napsty** [Napsty/check_es_system#39]
The plugin simply returns OK with an empty output when the Cluster is not there, for example when a "wrong" URL is specified using the `-H` parameter:

```
$ ./check_es_system.sh -H wrong-cluster-url.example.com -P 9243 -S -u user -p pass -t status
$ echo $?
0
```

Wrong URL in this case means that something responds but no stats can be read from it. In this particular case an outdated (archived?) Elasticsearch URL from Elastic.co was used. 

Debug (manually executing curl from the background) shows:

```
$ curl -k -s --max-time 30 --basic -u user:pass https://wrong-cluster-url.example.com:9243/_cluster/stats
{"ok":false,"message":"Unknown resource."}

```

![image](https://user-images.githubusercontent.com/3702203/122198860-ced02e00-ce99-11eb-80cb-97ae71b5b1af.png)

![image](https://user-images.githubusercontent.com/3702203/122198894-d8599600-ce99-11eb-8e0c-42553a764a4e.png)

**Comment by Napsty** [comment]
Something similar happens when using some URL which responds with HTTP 200:

```
$ ./check_es_system.sh -H www.claudiokuenzler.com -P 443 -S -u user -p secret -t status
parse error: Invalid numeric literal at line 1, column 10
parse error: Invalid numeric literal at line 1, column 10
parse error: Invalid numeric literal at line 1, column 10
parse error: Invalid numeric literal at line 1, column 10
parse error: Invalid numeric literal at line 1, column 10
parse error: Invalid numeric literal at line 1, column 10
parse error: Invalid numeric literal at line 1, column 10
parse error: Invalid numeric literal at line 1, column 10
parse error: Invalid numeric literal at line 1, column 10

$ echo $?
0
```

**Comment by Napsty** [comment]
Fixed in #40
